src/packets/ClientProtocol/ClientProtocol_1080/vehicle.ts

```diff
@@ -195,7 +195,20 @@ export const vehiclePackets: PacketStructures = [
       ]
     }
   ],
-  ["Vehicle.AccessType", 0x891c, {}],
+  [
+  "Vehicle.AccessType", 
+  0x891c,
+  {
+    fields: [
+      {
+        name: "vehicleGuid",
+        type: "uint64string",
+        defaultValue: "0x000000000000000000"
+      },
+      { name: "accessType", type: "uint16", defaultValue: 0 }
```

Comment:
What are the different accessTypes ?Llike 0 is open 2 is locked ? What is 1 and the other ones? Beside that it looks good will just test it in game and it will be merge, nice one ! :cowboy_hat_face:

Reply:
I should've been more specific, but yes from what I've tested it seems 2 is locked and every other number : 1, 3, 4, 5, 6, 7, 8, 9 are all unlocked. I couldn't find another use for the other numbers but it seems like 0 and 2 are the "main" numbers.

Reply:
Maybe one of the number also check the "team" the vehicle is on or smth like that ( since it's ps2 stuff )